components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { Text, Box, SimpleGrid, GridItem, Flex } from "@chakra-ui/react";
+import DaysOfWeekButton from "./DayOfTheWeekButton";
+import LinkButton from "../LinkButton/LinkButton";
+
+export default function DaysOfTheWeekContainer ({ setNumberOfDays, onSaveEvent, customHref }) {
+
+  // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
+  const [ daysOfTheWeek, setDaysOfTheWeek ] = useState([
+    {
+      id: 0,
+      day: "Monday",
+      isSelected: false
+    },
+    {
+      id: 1,
+      day: "Tuesday",
+      isSelected: false
+    },
+    {
+      id: 2,
+      day: "Wednesday",
+      isSelected: false
+    },
+    {
+      id: 3,
+      day: "Thursday",
+      isSelected: false
+    },
+    {
+      id: 4,
+      day: "Friday",
+      isSelected: false
+    },
+    {
+      id: 5,
+      day: "Saturday",
+      isSelected: false
+    },
+    {
+      id: 6,
+      day: "Sunday",
+      isSelected: false
+    }
+  ]);
+
+  // state to activate and disable save button (link button)
+  const [ saveButtonActive, setSaveButtonActive ] = useState(true);
+
+  // on click, the buttom will change colour and the state will be updated to include selected buttons
+  const handleClick = (e) => {
+    e.preventDefault();
+
+    let updatedData = [...daysOfTheWeek];
+
+    // based on the day name from innerHTML of each button, update the state if selected or unselected
+    updatedData.map(item => {
+      if (item.day === e.target.innerText) {
+        item.isSelected = item.isSelected ? false : true;
+      }
+    })
+    setDaysOfTheWeek(updatedData);
+
+    // activate Save button (link button) when something is selected
+    const areDaysSelected = updatedData.find(item => item.isSelected) ? false : true;
+    setSaveButtonActive(areDaysSelected);
+
+    // get answer to how many days a week user works
+    const workingDaysNumber = updatedData.filter(item => item.isSelected).length;
+    console.log(workingDaysNumber)
+    setNumberOfDays(workingDaysNumber)
+  }
+
+ // NOTE! I use minW attribute for the Box for now because at the moment the layout component has limitation for width. 
+ // Please change once Layout is changed
+
+  return (
+    <Box 
+      justify="center" 
+      borderWidth="2px" 
+      borderRadius="lg" 
+      minW="750px" 
+      py="8%"
+    >
+      <Flex 
+        px="20"
+        direction={"column"}
+      >
+        <Text>Select days of the week</Text>
```

Comment:
Should we make the font in the message "Select days of the week" heavier to match the Figma?
![image](https://user-images.githubusercontent.com/20964807/157336498-c6beb660-ef10-4aac-93f3-a03657c2bd4d.png)

Reply:
Happy to do that however to my understanding in Figma it's a regular font, not bold:
![image](https://user-images.githubusercontent.com/88268603/157342831-47440b3a-6931-455d-a254-229221fe2edf.png)
![image](https://user-images.githubusercontent.com/88268603/157342964-9e590b5b-894e-428b-b695-d6bd363e7d9a.png)

Reply:
am I looking at the wrong Figma version?

Reply:
Updated.